fitbenchmarking/fitting/mantid/tests/test_func_def.py

```diff
@@ -96,16 +96,16 @@ def create_wks_NIST_problem_without_errors(self):
         wks_exp = msapi.CreateWorkspace(DataX=prob.data_x, DataY=prob.data_y)
         return wks_exp
 
-    def test_functionDefinitions_return_NIST_functions(self):
+    # def test_functionDefinitions_return_NIST_functions(self):
 
-        prob = self.NIST_problem()
+    #     prob = self.NIST_problem()
```

Comment:
Please never include commented out code in production code. The control control keeps versions add changes can be rolled back. Please delete commented out code

Reply:
Sorry, I will go through the changes and make sure there is no commented code